Sources/Edge.swift

```diff
@@ -19,8 +19,7 @@ public class Edge: NSObject, Extension {
     private let LOG_TAG = "Edge" // Tag for logging
     private var networkService: EdgeNetworkService = EdgeNetworkService()
     private var networkResponseHandler: NetworkResponseHandler?
-    private var hitQueue: HitQueuing?
-    private var currentPrivacyStatus: PrivacyStatus?
+    internal var state: EdgeState?
```

Comment:
What's the reason for this being internal? Testing?

Reply:
overwriting it in EdgeExtensionTests